cli/cli.go

```diff
@@ -151,6 +155,20 @@ func preRun(cmd *cobra.Command, args []string) {
 	feedback.SetOut(colorable.NewColorableStdout())
 	feedback.SetErr(colorable.NewColorableStderr())
 
+	updaterMessageChan = make(chan *semver.Version)
+	go func() {
+		if cmd.Name() == "version" {
+			// The version command checks by itself if there's a new available version
+			updaterMessageChan <- nil
+		}
+		// Starts checking for updates
+		currentVersion, err := semver.Parse(globals.VersionInfo.VersionString)
```

Comment:
Is VersionString supposed to be always semver compliant? if yes you can use `semver.MustParse` to avoid checking the error

Reply:
It's not always semver compliant, it can also be `git-snapshot` or `nightly-<timestamp>` so I can't use `semver.MustParse`.

Reply:
Shouldn't those versions be changed to `0.0.0-git` or `0.0.0-nightly-<timestamp>`?
These non-semver versions plugged in feels "wrong" to me...

Also we already had:

```go
var (
	defaultVersionString = "0.0.0-git"
	versionString        = ""
	commit               = ""
	status               = "alpha"
	date                 = ""
	tr                   = i18n.Tr
)
```
in `version.go` IMHO we should follow the same pattern and force semver compliance.

Reply:
I get what you mean but probably would be better if done in a separate PR.